marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -768,22 +714,24 @@ instance Buffered UtxoHandle where
                inlineScript,
                inlineScriptHash,
                txIndexInBlock,
+               slotNo
+            ) VALUES
+            (?, ?, ?, ?, ?, ?, ?, ?, ?)|]
+            rows
+            `concurrently_` SQL.executeMany
+              c
+              [sql|INSERT OR IGNORE INTO blockInfo (
```

Comment:
The `OR IGNORE` part shouldn't be required, but the generators are wrong and it fails the tests without it.